packages/modules/pricing/src/repositories/pricing.ts

```diff
@@ -81,7 +81,7 @@ export class PricingRepository
         this.on("pl.id", "price.price_list_id").andOn(
           "pl.status",
           knex.raw("?", [PriceListStatus.ACTIVE])
-        )
+        ).andOn(knex.raw("pl.deleted_at IS NULL"))
```

Comment:
If you can add a similar where clause on the price table, that will be helpful!

Reply:
Sure I will do!

Reply:
[704f8c4](https://github.com/medusajs/medusa/pull/9015/commits/704f8c459085580c32a7fbdc46951066975d5d50)

I think this filters it, tested on production

Reply:
yup, that looks good! Thanks for the fix!